### PR TITLE
Remove sdl1 as a dependency of libwebp

### PR DESCRIPTION
**Pull request by rioki**
**Describe the pull request**

It may be historic but as things stand, sdl1 is not required to build libwebp on windows. It would be nice if someone maintaining the package could weigh in on that subject.

- What does your PR fix? Fixes issue #

sdl2 and sdl1 on the same installation force manual linking, which is undesired. As it appears that this dependency is defunct, removing the dependency keeps my installation free of sdl1

- Which triplets are supported/not supported? Have you updated the CI baseline?

I tested x86-windows and x64-windows locally. [the dependency was windows only]

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

I hope so...

### CODE REVIEW

**Comment by Neumann-A** [comment]
please see #9635. SDL is required for some options of the build so you cannot simply remove SDL as a build requirement for `all`

**Comment by rioki** [comment]
@Neumann-A I like your PR  #9635, since it kind of solves my problem and I really don't have stake in the issue. 

But the requirement, as I see it is you need SDL for emscripten build, which IMHO defies builds that target x86-window / x64-windows.

**Comment by LilyWangL** [comment]
@rioki, thanks for your PR! Can you please correct the Version field in the CONTROL file? You can see more detail from the maintainer guide.

File: ports/libwebp/CONTROL [hunk 1]
```diff
@@ -5,4 +5,4 @@ Description: Lossy compression of digital photographic images.
 
 Feature: all
 Description: enable all webp features
-Build-Depends: giflib, libjpeg-turbo, zlib, libpng, tiff, freeglut (!osx), sdl1 (windows), opengl
```
Comment (vicroms) [inline ports/libwebp/CONTROL]: Bump version to `1.0.2-9`

**Review by vicroms — CHANGES_REQUESTED** [review]

**Pull request closed without merging**